src/journal-remote/journal-remote.c

```diff
@@ -353,6 +361,9 @@ int journal_remote_server_init(
 void journal_remote_server_destroy(RemoteServer *s) {
         size_t i;
 
+        if (!s)
+            return;
```

Comment:
16spaces

Reply:
D'oh, who's the little rascal that keeps eating my spaces.